.github/workflows/regsync-config.yaml

```diff
@@ -14,11 +14,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      
-      - name: Generate Regsync Config
+
+      # Need to remove export version once rancher/charts gets the latest version 
+      # of charts-build-script binary.   
+      - name: Generate RegSync & Commit
         run: |
+          gh pr checkout ${{ github.event.pull_request.number }}
+          export CHARTS_BUILD_SCRIPT_VERSION=v0.4.2 
           make pull-scripts
           make regsync
+          git add regsync.yaml
+          git config --global user.name "Github Action RegSync"
+          git config --global user.email "github@action.com"
+          git commit -m "Updating regsync.yaml"
```

Comment:
I don't know how is the behavior of this running, but in the terminal `git branch` get's us to some sort of VIM interface.
Perhaps a good replacement for this right here would be:

`git rev-parse --abbrev-ref HEAD`

Reply:
Removed git branch